lope.go

```diff
@@ -77,22 +77,27 @@ func (l *lope) createDockerfile() {
 func (l *lope) addEnvVars() {
 	for _, e := range l.envs {
```

Comment:
Depending on the number of times cycles this goes through regexes can get expensive. Since the `blacklist` and `whitelist` are fixed at this point, you could `Compile()` them before this loop starts and use the compiled ones instead of `MatchString(re, name)` later.